shared/src/common/types/user/user-get-all-request-query.type.ts

```diff
@@ -0,0 +1,6 @@
+type UserGetAllRequestQueryDto = {
```

Comment:
```suggestion
type UsersGetAllRequestQueryDto = {
```
don't forget about file name